recipes/foldalign/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "2.5.1" %}
+{% set sha256 = "838075ee0e326baff3befc04dfb5777114466a7fb65408fa9af2cd63af096be5" %}
+
+package:
+  name: foldalign
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  fn: foldalign.-{{ version }}.tgz
+  url: https://rth.dk/resources/foldalign/software/foldalign.{{ version }}.tgz
+  sha256: {{ sha256 }}
+ 
+requirements:
+  build:
+    - gcc   # [linux]
```

Comment:
@mmiladi we changed this with cb3. Can you please look up most recent packages and adopt this section. There now needs to be a host section and as well.

Reply:
Sure thanks.